### PR TITLE
[WIP] OCPBUGS-35895: Enable fips for IPA

### PR DESCRIPTION
**Pull request by elfosardo**

### CODE REVIEW

**Comment by zaneb** [comment]
This appears to add `fips=1` unconditionally. Is that safe? I'd have thought we only want to enable it when the host cluster is in FIPS mode (i.e. `/proc/sys/crypto/fips_enabled` contains `1`).

**Comment by elfosardo** [comment]
forgot to add a WIP here, I wanted to see how the CI was doing first!

File: ironic-config/ironic.conf.j2 [hunk 1]
```diff
@@ -213,7 +213,7 @@ images_path = /shared/html/tmp
 instance_master_path = /shared/html/master_images
 tftp_master_path = /shared/tftpboot/master_images
 tftp_root = /shared/tftpboot
-kernel_append_params = nofb nomodeset vga=normal ipa-insecure=1 {% if env.IRONIC_RAMDISK_SSH_KEY %}sshkey="{{ env.IRONIC_RAMDISK_SSH_KEY|trim }}"{% endif %} {{ env.IRONIC_KERNEL_PARAMS|trim }} systemd.journald.forward_to_console=yes
+kernel_append_params = nofb nomodeset vga=normal ipa-insecure=1 fips=1 {% if env.IRONIC_RAMDISK_SSH_KEY %}sshkey="{{ env.IRONIC_RAMDISK_SSH_KEY|trim }}"{% endif %} {{ env.IRONIC_KERNEL_PARAMS|trim }} systemd.journald.forward_to_console=yes
```
[the remark this answers is not in the document]
Comment (elfosardo) [reply ironic-config/ironic.conf.j2:216]: yes, the idea is to make this conditional and modify upstream first
I was testing CI here and forgot to add a WIP

**Pull request closed without merging**